src/Game.js

```diff
@@ -0,0 +1,83 @@
+import Car from './Car.js';
+import InputView from './InputView.js';
+import OutputView from './OutputView.js';
+import { splitCarNames } from './utils/splitCarNames.js';
+import { validateCarName, validateRoundCount } from './validate.js';
+
+class Game {
+  #round;
+
+  constructor() {
+    this.cars = [];
+    this.#round = 0;
+  }
+
+  #createCars(inputCarNames) {
+    const carNames = splitCarNames(inputCarNames);
+
+    return carNames.map((carName) => new Car(carName));
+  }
+
+  #setValidatedValues(roundCount, inputCarNames) {
+    this.#round = roundCount;
+    this.cars = this.#createCars(inputCarNames);
+  }
+
+  async start() {
+    const inputCarNames = await InputView.readCarNames();
+    validateCarName(inputCarNames);
+    const roundCount = await InputView.readRoundCount();
+    validateRoundCount(roundCount);
+
+    this.#setValidatedValues(roundCount, inputCarNames);
+
+    OutputView.printEmptyLine();
+    this.#progress();
+  }
+
+  #raceCars() {
+    this.cars.map((car) => {
+      car.runRound();
+      const [name, forwardCount] = [car.name, car.forwardCount];
+      OutputView.printCurrentCarInfo(name, forwardCount);
+    });
+  }
+
+  #progress() {
+    let currentRound = 0;
+
+    while (currentRound < this.#round) {
+      if (currentRound === 0) OutputView.printExecutionResultMessage();
+
+      this.#raceCars();
+      OutputView.printEmptyLine();
+
+      currentRound += 1;
+    }
+
+    this.#end();
+  }
+
+  #end() {
+    const winners = this.getWinners();
+
+    OutputView.printWinners(winners);
+  }
+
+  #getCardAdvances() {
```

Comment:
`Card` 는`Car` 의 오타라고 생각이 들어서 체크했습니당 아니라면 무시하셔도 될 것 같아요

```suggestion
#getCarAdvances() {
```